crates/agentgateway-app/src/main.rs

```diff
@@ -0,0 +1,82 @@
+// Copyright Istio Authors
```

Comment:
There are a attributions from other projects throughout.  Given that you have changed these files, you might consider adding 

```
// Copyright agentgateway authors
```
(and/or solo.io) to the top of them all.